Assets/LeapMotionModules/Package/Editor/PackageDefinitionEditor.cs

```diff
@@ -0,0 +1,211 @@
+    }
+
+    private void drawFileElement(Rect rect, SerializedProperty property) {
+      drawExplorerElement(rect, property, EditorUtility.OpenFilePanel);
+    }
+
+    private void drawPackageElement(Rect rect, SerializedProperty property) {
+      EditorGUI.PropertyField(rect, property, GUIContent.none);
+    }
+
+    private ReorderableList createList(string propertyName, Action<Rect, SerializedProperty> drawMethod) {
+      SerializedProperty listProperty = serializedObject.FindProperty(propertyName);
+
+      var list = new ReorderableList(serializedObject, listProperty,
+                                     draggable: true,
+                                     displayHeader: true,
+                                     displayAddButton: true,
+                                     displayRemoveButton: true);
+
+      list.drawElementCallback += (rect, index, isActive, isFocused) => {
+        SerializedProperty property = list.serializedProperty.GetArrayElementAtIndex(index);
+        drawMethod(rect, property);
+      };
+
+      list.drawHeaderCallback += (rect) => {
+        GUI.Label(rect, listProperty.displayName);
+      };
+
+      list.elementHeight = EditorGUIUtility.singleLineHeight;
+
+
+      specifyCustomDrawer(propertyName, p => list.DoLayoutList());
+
+      return list;
+    }
+
+    private void drawExplorerElement(Rect rect, SerializedProperty property, Func<string, string, string, string> openAction) {
+      Rect leftRect = rect;
+      Rect rightRect = rect;
+
+      leftRect.width -= 100;
+
+      rightRect.x = leftRect.x + leftRect.width;
+      rightRect.width = 100;
+
+      EditorGUI.TextField(leftRect, property.stringValue);
+
+      if (GUI.Button(rightRect, "Change")) {
+        string chosenFolder = openAction("Select Folder", Application.dataPath, "");
+        if (!string.IsNullOrEmpty(chosenFolder)) {
+          string relativePath = makeRelativePath(Application.dataPath, chosenFolder);
+          if (!string.IsNullOrEmpty(relativePath) && !relativePath.StartsWith("..")) {
+            property.stringValue = relativePath;
+          }
+        }
+      }
+    }
+
+    private static string makeRelativePath(string fromPath, string toPath) {
+      if (string.IsNullOrEmpty(fromPath)) throw new ArgumentNullException("fromPath");
+      if (string.IsNullOrEmpty(toPath)) throw new ArgumentNullException("toPath");
+
+      Uri fromUri = new Uri(fromPath);
+      Uri toUri = new Uri(toPath);
+
+      if (fromUri.Scheme != toUri.Scheme) { return toPath; } // path can't be made relative.
+
+      Uri relativeUri = fromUri.MakeRelativeUri(toUri);
+      string relativePath = Uri.UnescapeDataString(relativeUri.ToString());
+
+      if (toUri.Scheme.Equals("file", StringComparison.InvariantCultureIgnoreCase)) {
+        relativePath = relativePath.Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar);
+      }
+
+      return relativePath;
+    }
+
+    private void generateBuildMenuScript() {
```

Comment:
It tickles my brain that the menu script generator is shorter than the generated script...